requirements.txt

```diff
@@ -2,7 +2,7 @@ backoff
 docker < 4.3.0
 docker-squash>=1.0.7
 dockerfile-parse>=0.0.13
-flatpak-module-tools >= 0.11,<0.13;python_version<"3.9"
+flatpak-module-tools >= 0.11,<=0.13;python_version<"3.9"
```

Comment:
0.13 does only work with python>=3.9, it was fixed later in 0.13.1

Reply:
well 0.13 in pypi isn't fixed for 3.9, 0.13 fedora rpm is :(